stats.go

```diff
@@ -0,0 +1,386 @@
+/*
+   Copyright (C) nerdctl authors.
```

Comment:
This file seems derivative of containerd, right?

Reply:
not 100%, several logic I introduced them myself

Reply:
You want to add containerd Copyright ?

Reply:
Yes